dockers/docker-fpm-frr/frr/supervisord/supervisord.conf.j2

```diff
@@ -39,6 +39,17 @@ stderr_logfile=syslog
 dependent_startup=true
 dependent_startup_wait_for=rsyslogd:running
 
+[program:zsocket]
+command=/usr/bin/zsocket.sh
```

Comment:
@shi-su , i think the problem you discovered is a generic problem. i think it is better that you pull you script as a delicated script and wait for zebra socket available, and then both bgpd, staticd or other daemons shoud wait for that script to exit.

Reply:
Agree. I will rewrite my script as a separate one to wait for zebra to be available and then let the daemons wait for it to exit similar to the code changes here.

Reply:
I drafted a PR to pull the script https://github.com/Azure/sonic-buildimage/pull/6519. It took some ideas in this PR, thanks to @sudhanshukumar22. Maybe we can stick to that script. There are some other changes in this PR, maybe they can be rebased onto https://github.com/Azure/sonic-buildimage/pull/6519.

Reply:
@ben-gale  to comment

Reply:
I will review that code.

Reply:
Since @shi-su has done the changes for wait script, I will remove this. I will put my changes on top of them.